hudi-common/src/main/java/org/apache/hudi/avro/MercifulJsonConverter.java

```diff
@@ -245,10 +245,14 @@ private static JsonToAvroFieldProcessor generateFixedTypeHandler() {
     return new JsonToAvroFieldProcessor() {
       @Override
       public Pair<Boolean, Object> convert(Object value, String name, Schema schema) {
-        byte[] src = value.toString().getBytes();
+        ArrayList<Integer> converval = (ArrayList<Integer>) value;
```

Comment:
Minor Comment: Can you use List<Integer> instead of ArrayList<Integer>

Reply:
Sure

Reply:
> Minor Comment: Can you use List instead of ArrayList

Done